Makefile

```diff
@@ -292,6 +292,26 @@ ifeq ($(runtime-bitcode),yes)
   endif
 endif
 
+# default OPENSSL version
+ifdef default_openssl
+  BUILD_FLAGS += -DPONY_DEFAULT_OPENSSL=\"$(default_openssl)\"
+  OPENSSL=-D$(default_openssl)
+endif
+
+ifneq (,$(OPENSSL))
+  ifeq ("-Dopenssl_0.9.0","$(OPENSSL)")
+    OPENSSL_VALID:=ok
+  endif
+  ifeq ("-Dopenssl_1.1.0","$(OPENSSL)")
+    OPENSSL_VALID:=ok
+  endif
+  ifeq (ok,$(OPENSSL_VALID))
+    $(warning targeting openssl $(OPENSSL))
+  else
+    $(error OPENSSL=$(OPENSSL) is invalid, expecting one of -Dopenssl_0.9.0 or -Dopenssl_1.1.0)
```

Comment:
This is likely to be confusing if someone specifies an invalid `default_openssl` value but the error message says that `OPENSSL` is invalid.

Reply:
Good point, fixing.